CONTRIBUTING.md

```diff
@@ -48,285 +59,66 @@ Some examples of "Architecture" improvements:
 
 ```sh
-git push -f origin myfeature
+export GOPATH=$HOME/go
+export PATH=$PATH:$GOPATH/bin
 ```
 
-### Step 8: Create a pull request
-
-1. Visit your fork at https://github.com/$user/tidb (replace `$user` obviously).
-2. Click the `Compare & pull request` button next to your `myfeature` branch.
-
-### Step 9: Get a code review
+#### Dependency management
 
-Once your pull request has been opened, it will be assigned to at least two
-reviewers. Those reviewers will do a thorough code review, looking for
-correctness, bugs, opportunities for improvement, documentation and comments,
-and style.
+TiDB uses [`Go Modules`](https://github.com/golang/go/wiki/Modules) to manage dependencies.
 
-Commit changes made in response to review comments to the same branch on your
-fork.
+## Workflow
 
-Very small PRs are easy to review. Very large PRs are very difficult to
-review.
+See the [Github Workflow](https://github.com/pingcap/community/blob/master/contributors/workflow.md)
 
 ## Code style
 
-The coding style suggested by the Golang community is used in TiDB. See the [style doc](https://github.com/golang/go/wiki/CodeReviewComments) for details.
-
-### Code comment style
-
-Follow this style to write TiDB code comment.
-
-For the code comment style of the TiKV project, see [Code Comment Style](https://github.com/tikv/tikv/blob/master/CODE_COMMENT_STYLE.md).
-
-**Format:**
-
-- Use `//` for a single-line comment and trailing comment.
-- Use `/* ... */` for a block comment (used only when needed).
-- Use **gofmt** to format your code.
-- Place the single-line or block comment above the code it's annotating.
-- Fold long lines of comments.
-- The maximum width for a line is 100 characters.
-- For a comment containing a URL
-
-    - Use a **relative URL** if the text is linked to a file within the same GitHub repository.
-    - Use an **absolute URL** in **docs and docs-cn repositories** if the code with this comment is copied from another repository.
-
-**Language:**
-
-- Word
-    
-    - Use **American English** rather than British English.
-    - Use correct spelling.
-    - Use **standard or official capitalization**.
-    - Use words and expressions consistently.
-    - Do not use lengthy compound words.
-    - Do not abbreviate unless it is absolutely necessary.
-    - "we" should be used only when it means the code writer *and* the reader.
-
-- Sentence
-
-    - Use standard grammar and correct punctuation.
-    - Use relatively short sentences.
-
-- Capitalize the first letter of sentences and end them with periods.
-    
-    - If a lower-case identifier comes at the beginning of a sentence, don't capitalize it.
-
-- When used for description, comments should be **descriptive** rather than **imperative**.       
-
-- Use "this" instead of "the" to refer to the current thing.
-
-- The Markdown format is allowed.
-
-For more details, see [Code Comment Style](code-comment-style.md).
-
-## Commit message style
```

Comment:
Is there any place else has this kind of content?

Reply:
I think it’s covered by the “Commit Message and Pull Request Style” section, although the content isn’t the same.